src/context/AppProvider/AppContext.tsx

```diff
@@ -134,7 +138,7 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
           const { hasActivity } = account.accounts.byId[accountId]
 
           // don't add accounts with no activity past account 0
-          if (accountNumber > 0 && !hasActivity) return
+          if (accountNumber > 0 && !hasActivity) return delete accountMetadataByAccountId[accountId]
```

Comment:
The guts of the mismatch. We *should* delete metadata for accounts without activity.